acceptance-tests/src/test/resources/features/cms/preview.feature

```diff
@@ -0,0 +1,8 @@
+Feature: CMS Preview
+    @WIP
```

Comment:
Is the annotation left in place intentionally?

Reply:
Nope, thanks 😁